### PR TITLE
replace inline clear password by environment variable

### LINKED ISSUE #1668: java_keystore: passwords are passed as command line arguments to Java's keytool
##### SUMMARY
As topic says. Should be changed to use stdin to pass the passwords in (same as for the `openssl` calls, see #276 for that).

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
java_keystore

##### ANSIBLE VERSION
```paste below
all
```

### PR DESCRIPTION
**Pull request by quidame**
##### SUMMARY

This PR covers secret's handling by the module. It replaces insecure and clear password in keytool's commandline arguments by storing it in an environment variable (doing this from stdin is not supported by keytool).

This is achieved with the `environ_update` argument of `module.run_command()` (i.e. on a per-command basis) rather than with a more general `module.run_command_environ_update()` (that would export the variable in all subsequent `module.run_command()` calls, that means: when it is not needed).

Fixes #1668 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

java_keytool

### CODE REVIEW

File: changelogs/fragments/2177-java_keystore_1668_dont_expose_secrets_on_cmdline.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
```
Comment (felixfontein) [inline changelogs/fragments/2177-java_keystore_1668_dont_expose_secrets_on_cmdline.yml:2]: I think it is ok to label this as what it is:
```suggestion
security_fixes:
```

**Review by felixfontein — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@quidame thanks a lot for fixing this!

**Comment by Andersson007** [comment]
@quidame it's a bit out of topic, i'd like to ask you a couple of questions about contributing to Ansible in general. If you don't mind, could you please send me a test email to aklychko at redhat dot com? (i didn't find yours in your profile, otherwise I would've done it myself:)

**Pull request merged**